include/deal.II/fe/fe_enriched.h

```diff
@@ -148,7 +148,9 @@ DEAL_II_NAMESPACE_OPEN
  @Article{Davydov2016,
    Title                    = {On the h-adaptive PUM and hp-adaptive FEM approaches applied to PDEs in quantum mechanics.},
    Author                   = {Davydov, D and Gerasimov, T and Pelteret, J.-P. and Steinmann, P.},
-   Journal                  = {Journal of Computational Physics. Submitted},
+   archivePrefix            = {arXiv},
+   eprint                   = {1612.02305},
+   primaryClass             = {physics.comp-ph},
```

Comment:
If you do change the above, then this could change to `eprintclass` for consistency.